internal/buffer/unbounded_test.go

```diff
@@ -119,3 +119,14 @@ func (s) TestMultipleWriters(t *testing.T) {
 		t.Errorf("reads: %#v, wantReads: %#v", reads, wantReads)
 	}
 }
+
+func (s) TestClose(t *testing.T) {
+	ub := NewUnbounded()
+	ub.Close()
+	ub.Put(1)
+	ub.Load()
```

Comment:
What are these two operations testing? Right now, it's only asserting that this doesn't block and successfully returns. Could we add an assertion that nothing should get sent on the Get() channel as the result of these two operations due to the Close() operation previous? (this is tested as part of the ub.Get = ok conditional below, but I think since you do another Close() operation you should assert this at this point).